package.json

```diff
@@ -18,10 +18,8 @@
         "eslint-config-prettier": "^8.3.0",
         "eslint-plugin-prettier": "^4.0.0",
         "express": "^4.18.2",
-        "husky": "^7.0.4",
         "jest": "^27.2.0",
         "prettier": "^2.5.1",
-        "pretty-quick": "^3.1.3",
```

Comment:
figured a "quicker" `prettier` wrapper isn't necessary given the small amount of code in this repo and since it's being checked out of band in CI.

the idea is to allow the dev to decide what tooling they want to use to lint/format outside of CI, while we provide a more opinionated dev container with that tooling to execute on save (if using vs code).